config/maintainers.json

```diff
@@ -1,6 +1,16 @@
 {
   "docs_url": "https://github.com/exercism/docs/blob/master/maintaining-a-track/maintainer-configuration.md",
   "maintainers": [
+    {
```

Comment:
Can you also move yours to the bottom of the file <3 ?

Reply:
Got it!